app/bundles/LeadBundle/Form/Type/LeadType.php

```diff
@@ -194,6 +194,37 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
                     $builder->add($alias, $type, $opts);
                     break;
+                case 'multiselect':
```

Comment:
Is a new case needed here? Could we simply add it after `case 'boolean':` and change `if ($type == 'select' && !empty($properties['list'])) {` to `if (in_array($type, ['select', 'multiselect']) && !empty($properties['list'])) {` then `'multiple'    => false,` within that case to `'multiple'    => ('multiselect' === $type),`